third_party/istio-1.0-prerelease/istio.yaml

```diff
@@ -672,21 +1109,13 @@ data:
             - NET_ADMIN
           privileged: true
         restartPolicy: Always
-
+      
       containers:
       - name: istio-proxy
-        # PATCH #2:  Add a prestop sleep.
```

Comment:
Why do we not need this patch in 1.0 ?

Reply:
Good catch.  I am adding that, with a TODO to remove once we fix #1370.